pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -225,6 +226,28 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
```

Comment:
we should also watch for Constraint CRDs changes only when operation generate is enabled.